starknet_py/tests/e2e/conftest.py

```diff
@@ -52,6 +52,9 @@
     "0X126FAB6AE8ACA83E2DD00B92F94F3402397D527798E18DC28D76B7740638D23"
 )
 
+directory_with_contracts = Path(os.path.dirname(__file__)) / "mock_contracts_dir"
+mocks_dir = Path(os.path.dirname(__file__)) / "mocks"
```

Comment:
If you want to name your directory `mocks` perhaps we should consider renaming `mock_contracts_dir` to just `contracts`?

Reply:
cc @war-in

Reply:
I just think having `mocks/` and `mock_contracts_dir/` is kind of confusing. For me it's fine to have "mock" contracts in `contracts/`, but if you don't like it, we don't have to go with that approach.